### ISSUE TITLE
Example build issue - missing local.properties

### ISSUE THREAD

**Issue by csoma** [dropbox/djinni#8]
Error message on Mac:

> NDK is not configured. Make sure there is a local.properties file with an ndk.dir entry in the directory

Following the instructions in [README.md](https://github.com/dropbox/djinni/blob/master/example/README.md) I get these errors:

```
$ make android
make: *** No rule to make target `android'.  Stop.

$ make ios
make: *** No rule to make target `ios'.  Stop.

$ make example_android
...

cd example/android/ && ./gradlew app:assembleDebug

FAILURE: Build failed with an exception.

* Where:
Build file 'djinni/example/android/app/build.gradle' line: 38

* What went wrong:
A problem occurred evaluating project ':app'.
> NDK is not configured. Make sure there is a local.properties file with an ndk.dir entry in the directory djinni/example/android.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

Suggestions:

Since `local.properties` is excluded in gitignore, creating a `local.properties.sample` would be a great help.

Also updating the [README.md](https://github.com/dropbox/djinni/blob/master/example/README.md) with the steps required to download and install SDK/NDK would allow prospective users to quickly compile & test.

**Comment by mbrgm** [comment]
@csoma Can this be closed?

**Comment by csoma** [comment]
Yes, I found a workaround.

The proposed solution (creating `local.properties.sample` + documentation on how to set it up) would help others...

**Comment by j4cbo** [comment]
Fixed by #50
